acceptancetests/assess_cross_model_relations.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python
+"""Functional tests for Cross Model Relation (CMR) functionality.
+
+This test exercises the CMR Juju functionality which allows applications to
+communicate between different models (including across controllers/clouds).
+
+The outline of this feature can be found here[1].
+
+This test will exercise the following aspects:
+  - Ensure a user is able to create an offer of a relation of an application
+    including:
```

Comment:
The 3 items below are important, but the main thing we need to test should be explicitly called out also - that a user is able to consume and relate to a given offer